src/actions.ts

```diff
@@ -0,0 +1,38 @@
+// TODO: This is a copy of this: https://github.com/redux-observable/redux-observable/blob/master/src/ActionsObservable.js
+// Remove after this is resolved: https://github.com/redux-observable/redux-observable/issues/95
+import { Injectable, Inject } from '@angular/core';
+import { Action, Dispatcher } from '@ngrx/store';
+import { Observable } from 'rxjs/Observable';
+import { Operator } from 'rxjs/Operator';
```

Comment:
I haven't fully implemented the typings for the `lift` method and importing the `Operator` interface will be required to do so. See: https://github.com/ngrx/store/blob/master/src/store.ts#L25